FifthLesson/VectorProgram/VectorProgramTests/ValidatorTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using VectorProgram.Controller;
+using VectorProgram.Model;
+
+namespace VectorProgramTests
+{
+    [TestClass]
+    public class ValidatorTests
+    {
+        [TestMethod]
+        public void ValidateInput_CorrectValues()
+        {
+            bool actual;
+
+            var inputMultiplier = new[] { "1", "56565", "1.0", "1,54" };
+            var inputVector = new[] { "0,005 0,005 0,005", "0.549 0.549 0.549", "1 1 1" };
+
+            foreach (var item in inputMultiplier)
+            {
+                actual = Validator.ValidateInput(DataType.Multiplier, item);
+                Assert.AreEqual(true, actual);
+            }
+
+            foreach (var item in inputVector)
+            {
+                actual = Validator.ValidateInput(DataType.Multiplier, item);
+                Assert.AreEqual(true, actual);
+            }
+        }
+
+        [TestMethod]
+        public void ValidateInput_WrongValues()
+        {
+            bool actual;
+
+            var inputMultiplier = new[] { string.Empty, "asf" };
+            var inputVector = new[] { "kl", string.Empty };
+
+            foreach (var item in inputMultiplier)
+            {
+                actual = Validator.ValidateInput(DataType.Multiplier, item);
+                Assert.AreEqual(false, actual);
+            }
+
+            foreach (var item in inputVector)
+            {
+                actual = Validator.ValidateInput(DataType.Multiplier, item);
```

Comment:
There should be _DataType.Vector_, shouldn't it?

Reply:
This has already been fixed.